test/parallel/test-http2-compat-serverresponse-trailers.js

```diff
@@ -26,17 +26,17 @@ server.listen(0, common.mustCall(() => {
     common.expectsError(
       () => response.setTrailer('test', undefined),
       {
-        code: 'ERR_HTTP2_INVALID_HEADER_VALUE',
```

Comment:
I'm not 100% sure about this because the semantics of the http2 validation are slightly different, for example pseudo headers are an http2 only feature.